recipes/stacks/1.37/meta.yaml

```diff
@@ -0,0 +1,27 @@
+package:
+  name: stacks
+  version: "1.37"
+build:
+  number: 1
+  skip: True # [osx]
```

Comment:
Oh I see it's copied and already exists. In this case leave it like it is. Just saying for the next recipe the `# [linux]` would be nice and you should adjust the 1.40 version to it.

Thanks!